bin/polymer.js

```diff
@@ -22,11 +22,12 @@ process.title = 'polymer';
 
 // NOTE 04-21-2017: Confirmed "semver" supports Node versions as low as 0.10
 var semver = require('semver');
+var version = require('../package.json').engines.node;
 
 // Exit early if the user's node version is too low.
-if (!semver.satisfies(process.version, '>=6')) {
+if (!semver.satisfies(process.version, version)) {
   console.log(
-      'Polymer CLI requires at least Node v6. ' +
+      'Polymer CLI requires at least Node v' + verion.replace(/[^\d\.]*/, '') + '. '
```

Comment:
Can you store the `version.replace` result in a variable and comment what it's doing?